_sass/_variables.scss

```
@@ -8,6 +8,9 @@ $color-nav: #444;
 $color-spacer: #eee;
 $color-submenu-bg: white;
 $color-bg: #fafafa;
```

Continuation of https://github.com/tc39/tc39.github.io/pull/201#discussion_r456735261&hellip;

To be technically correct here, while passing AA, we are going to need to change the background color from `#fafafa` to `#f5f5f5`, which would allow us to use that link foreground while passing.

Proof: https://webaim.org/resources/contrastchecker/?fcolor=017e78&bcolor=f5f5f5&api

```suggestion
$color-bg: #f5f5f5;
```